source/documentation/tools/create-a-derived-table/project-structure.md

```diff
@@ -1,66 +1,723 @@
+      │          ├── stg_xhibit__stg_<model_names>.sql
+      │          └── stg_xhibit__stg_court.sql
+      ├── courts
+      │   ├── courts_int
+      │   │
+      │   ├── courts_datamarts
+      │   │
+      │   └── criminal_courts_derived
+      │
+      ├── prison  # domain
+      │   ├── prison_int # database
+      │   │      ├──prison_int__int_<model_names>.sql # model
+      │   │      ...
+      │   ├── prison_datamarts
+      │   │
+
+```
+
+### Models
+
+Finally, you will begin designing / making your models. We suggest a medallion architecture that categorises the model you are making into three distinct layers, `staging`, `intermediate`, `datamarts` and `derived`. Where `datamarts` and `derived` are two halves of the same layer. Each layer represents data that is increasingly well understood, cleaned and standardised. 
+
+The general rule as to what goes into each of these layers comes down to a few considerations. Staging generally is source specific and doesn't change the grain of the data. Then when you want to combine different sources, or change the grain of the data (or other more complicated transformations) then the models belongs in an intermediate database. Finally, the user facing models live in either `_datamarts` or `_derived` databases. The following sections will go into more depth about these categorisations as well as what the medallion architecture is. 
+
+These concepts can be confusing, so if you have any further questions or just want to discuss you project use the [#ask-data-modelling](https://moj.enterprise.slack.com/archives/C03J21VFHQ9) slack channel to reach out. We are happy to answer questions or receive feedback about this guide there!
+
+Finally, it is worth flagging that create-a-derived-table has been evolving since its conception and for that reason project structure guidance, style guidance and best practice have all changed on several occasions. So when you start looking around the code base on create-a-derived-table you may find that code style, project structure or naming conventions don't match the guidance here. It will be a continuos effort from all that use it to slowly conform create-a-derived-table.
+
+## 2-staging
+
+The staging layer is where our journey begins. This is the foundation of our project, where we bring all the individual components we're going to use to build our more complex and useful models into the project.
+
+Dbt likes to use an analogy throughout this guide, and we will stick to it as we think it is useful; thinking modularly in terms of atoms, molecules, and more complex outputs like proteins or cells (we apologise in advance to any chemists or biologists for our inevitable overstretching of this metaphor). Within that framework, if our source system data is a soup of raw energy and quarks, then you can think of the staging layer as condensing and refining this material into the individual atoms we’ll later build more intricate and useful structures with.
```

Comment:
I think this is more confusing than helpful from dbt - it assumes an unnecessary level of scientific knowledge. If we want a metaphor then a better one (in my opinion is):
- sources as raw ingredients 
- staging is our initial preparation - for example chopping the raw ingredient
- intermediate is where we start combining the ingredients for a specific recipe
- dimensional layer is the key components that can be combined in different ways to create a range of meals
- derived is a pre-prepared meal created with a specific combination our key components

There's a decent example with toasted sandwiches here: https://www.lightdash.com/blogpost/bi-for-the-semantic-layer